### PR TITLE
LPD-24076 Add Data Migration Center as an unsupported application by Publications

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPD-24076](https://liferay.atlassian.net/browse/LPD-24076)

### CODE REVIEW

**Comment by david-truong** [comment]
@xbrianlee did we want to add it to the unsupported list or the production only list?

File: modules/test/playwright/tests/change-tracking-web/unsupportedPublicationsScope.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import {expect, mergeTests} from '@playwright/test';
+
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {changeTrackingPagesTest} from '../../fixtures/changeTrackingPagesTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+
+export const test = mergeTests(
+	featureFlagsTest({
+		'LPS-164563': true,
+	}),
```
Comment (pei-jung) [inline modules/test/playwright/tests/change-tracking-web/unsupportedPublicationsScope.spec.ts:15]: This is not the right feature flag. The original feature flag was removed in https://github.com/liferay/liferay-portal/commit/fb9b2167610f030692a3e91521f0356f425d62be, so we don't need to set it.

**Comment by david-truong** [comment]
ci:test:relevant

File: modules/test/playwright/tests/change-tracking-web/unsupportedPublicationsScope.spec.ts
Comment (pei-jung) [inline modules/test/playwright/tests/change-tracking-web/unsupportedPublicationsScope.spec.ts:1]: Rename file to something more descriptive. Maybe `unsupportedApplicationNotification`?

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
ci:test:stable

**Comment by pei-jung** [comment]
ci:test:stable

**Comment by pei-jung** [comment]
ci:forward:force

**Pull request closed without merging**